test/indicators/get_indicators.js

```diff
@@ -0,0 +1,53 @@
+const test = require('ava');
+
+const server = require('../../server');
+
+test.before(t => {
+  server.register(require('inject-then'), function (err) {
+    if (err) throw err;
+  });
+});
+
+test('get all indicators, no token', t => {
+  return server.injectThen('/indicators')
+    .then((res) => {
+      t.is(res.statusCode, 200, 'Status code is 200');
+      t.true(res.result.every(indicator => {
+        return indicator.name === 'public and published';
```

Comment:
@drewbo one way to do it would be to get all the data with the admin rights, and then make sure that the ones we get using the no token call are public and published. Another way is to inject the total fixture attributes for all the tests and compare against those: the server response is measured against the expected fixture data.